Tasks/InstallAppleProvisioningProfileV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Install Apple provisioning profile",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=862068)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=862068)",
```

Comment:
New strings just need to be updated in the task.json and task.loc.json files

Reply:
Solved in https://github.com/microsoft/azure-pipelines-tasks/pull/12917/commits/caf3be13d4e01917ebc037bcbe98d117f49c8080